src/main/resources/config/liquibase/changelog/20250409003009_changelog.xml

```diff
@@ -64,7 +64,10 @@
 
     <changeSet id="20250409-2-add-columns-with-differing-format-mysql" author="florian-glombik">
         <preConditions onFail="MARK_RAN">
-            <dbms type="mysql"/>
+            <or>
+                <dbms type="mysql"/>
+                <dbms type="h2"/>
```

Comment:
`h2` is the database type for the in memory MySQL database for integration tests